libs/platform/core/src/services/entity/entity-provider.ts

```diff
@@ -0,0 +1,34 @@
+import { Provider } from '@spryker-oryx/di';
+
+export const EntityProvider = 'oryx.EntityProvider*';
+
+export interface EntityProvider<E = unknown, Q = unknown> {
+  service: string;
+  // factory?: () => E;
+  context?: string;
+  // contextToQualifier?: () => Q;
```

Comment:
left overs?

Reply:
adjusted implementation to cover more flexibility